Dockerfile

```diff
@@ -0,0 +1,17 @@
+#
+# Build stage
+#
+FROM maven:3.6.0-jdk-11-slim AS build
+WORKDIR /usr/app
+COPY src ./src
+COPY pom.xml .
+RUN mvn clean install
+#
+# Package stage
+#
+FROM openjdk:11-jdk-slim
+ARG path=/usr/app
```

Comment:
ARG is similar to java constants, so should be uppercase

Reply:
fixed